libs/go/zmscli/cli.go

```diff
@@ -1017,6 +1017,14 @@ func (cli *Zms) EvalCommand(params []string) (*string, error) {
 				}
 				return cli.SetGroupReviewEnabled(dn, args[0], reviewEnabled)
 			}
+		case "set-group-delete-protection":
+			if argc == 2 {
+				deleteProtection, err := strconv.ParseBool(args[1])
+				if err != nil {
+					return nil, err
+				}
+				return cli.SetGroupReviewEnabled(dn, args[0], deleteProtection)
```

Comment:
wrong function is being called. it should be SetGroupDeleteProtection

Reply:
copy&paste is a terrible thing...
fixed